operations/resources.md

```diff
@@ -211,13 +211,13 @@ Returns all resources of an enterprise associated with the connector integration
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 
-#### Resource category
+#### Resource category - DEPRECATED
```

Comment:
@MikeAdamsMews I had a problem that I have two objects at the same place, one old and a new one. We should probably unify our approach and update style guide with it.

Reply:
@VBuben Following our convention, as this is `resourceCategories/getAll` then it should go in a new page `resourcecategories.md` not in `resources.md`. In that case, your problem with duplicate object definitions probably won't arise. Don't forget to add an entry in `SUMMARY.md` for the new page. In the Operations home page the new operation can be listed under theme `Enterprises` just like the other resource-related operations. Oh, and one more thing, we're in May now (Changelog 😉).

Reply:
@MikeAdamsMews Thanks, fixed!